pkg/buildpack/multi_architecture_helper.go

```diff
@@ -56,9 +59,16 @@ func (m *MultiArchConfig) CopyConfigFiles(baseDir string) ([]string, error) {
 
 // CopyConfigFile will copy the buildpack.toml file from the base directory into the corresponding platform folder
 // for the specified target and desired distribution version.
-func CopyConfigFile(baseDir string, target dist.Target) (string, error) {
+func CopyConfigFile(baseDir string, target dist.Target, buildpackType string) (string, error) {
+	var path string
+	var err error
+
 	if ok, platformRootFolder := PlatformRootFolder(baseDir, target); ok {
-		path, err := copyBuildpackTOML(baseDir, platformRootFolder)
+		if buildpackType == "extension" {
```

Comment:
```suggestion
		if buildpackType == KindExtension {
```
I think we can reuse some constants we already have defined [here](https://github.com/buildpacks/pack/blob/main/pkg/buildpack/buildpack.go#L21-L24)